### DISCUSSION TITLE
var not found

### DISCUSSION THREAD

**Discussion by Marty56** [arendst/Tasmota#15166] [Q&A] [answered]
Hi,

I have a simple script in which I want to access the values of my meters.
```
>D
pv=0
au=0

>B
=>sensor53 r
tper=30

>S
pv=PV#Lieferung

>M 1
+1,3,s,0,9600,PV
1,77070100020800FF@1000,Lieferung,kWh,Lieferung,2
#
```

unfortunately I am getting an error message
```
06:25:43.144 var not found: PV#Lieferung
```
if I am removing the line 'au=0' then error is gone
I have no glue what I am doing wrong.

regards Martin

**Comment by gemu2015** [comment, marked as answer]
you try to access a tele period JSON payload. this must be done in the >T not in >S section.

**Comment by Marty56** [comment]
Thanks for the feedback.
I had picked the >S Sektion because I do not really understand the >T documentation
"Executed at least at [TelePeriod](https://tasmota.github.io/docs/Commands/#teleperiod) time (SENSOR and STATE) but mostly faster up to every 100 ms, only put tele- vars in this section."

When is this section executed?  I understand "at least at Teleperiod". Ok, but the rest of the sentence is totally unclear for me.
